ee/fe/modules/340-extended-monitoring/monitoring/prometheus-rules/certificates/certificate.yaml

```diff
@@ -0,0 +1,29 @@
+- name: kubernetes.certmanager.certificate
```

Comment:
I would like to recommend adding the `group by` rules for the following alerts (plk_grouped_by__main annotation). You can take a look for samples in the `d8.extended-monitoring.cert-exporter.availability` alerts group from your PR.